### PR TITLE
docs: doc-755 added public IP addresses

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR publishes the Public Palette IP addresses that customers must know when configuring their Firewall rules.

CleanShot 2023-04-24 at 07 43 31

## Review Changes

💻 [Preview](https://deploy-preview-1256--docs-spectrocloud.netlify.app/architecture/palette-public-ips)

🎫 [DOC-755](https://spectrocloud.atlassian.net/browse/DOC-755)

### CODE REVIEW

File: content/docs/02.5-architecture/04.5-palette-public-ips.md
Comment (ritawatson) [inline content/docs/02.5-architecture/04.5-palette-public-ips.md:12]:
```suggestion
 In this section, you can find the public IP addresses that support Spectro Cloud SaaS operations. These IP addresses are essential to ensure seamless communication between your infrastructure and our platform.
```

Comment (ritawatson) [reply content/docs/02.5-architecture/04.5-palette-public-ips.md:12]: @karl-cardenas-coding The sentence is fine as is, and this very minor suggestion is totally up to you @karl-cardenas-coding.

**Review by ritawatson — APPROVED** [review]
LGTM - one minor suggestion if you choose to take it.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1256#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1256#pullrequestreview-1398555394" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-4/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
